Trouble including the editor as a sub-package in an AEMaaCS project

Hello,

I'm trying to get the Context Aware Configuration Editor installed in AEMaaCS by adding it to my `pom.xml` and having it installed with the `filevault-package-maven-plugin`.

I define a module in my project to gather all 3rd party tools and get them installed.

```
<plugin>
    <groupId>org.apache.jackrabbit</groupId>
    <artifactId>filevault-package-maven-plugin</artifactId>
    <extensions>true</extensions>
    <configuration>
        <name>${contentPackage.name}</name>
        <group>${contentPackage.group}</group>
        <packageType>container</packageType>
        <filterSource>src/main/content/META-INF/vault/filter.xml</filterSource>
        <subPackages>
            <subPackage>
                <groupId>io.wcm</groupId>
                <artifactId>io.wcm.caconfig.editor.package</artifactId>
                <filter>true</filter>
            </subPackage>
        </subPackages>
        <embeddedTarget>/apps/cp-aemaacs-prerequisites/install</embeddedTarget>
        <embeddeds>
            <embedded>
                <groupId>io.wcm</groupId>
                <artifactId>io.wcm.caconfig.extensions</artifactId>
            </embedded>
        </embeddeds>
        <allowIndexDefinitions>true</allowIndexDefinitions>
    </configuration>
</plugin>
```

The package is of type `container`. I've done something similar before, on an AEM 6.5 project, with an earlier version of the plugin and the Editor, and it worked fine.

Now I'm getting an error, and a warning at build time, both complaining about the sub-package type.
```
[INFO] Start validating sub package '/home/username/repos/foobarbaz/aem_cloud/prerequisites/target/prerequisites-1.0.0-SNAPSHOT.zip/jcr_root/etc/packages/wcm-io/io.wcm.caconfig.editor.package-1.15.6.zip'...
[INFO] Using 10 validators for package of type MIXED: jackrabbit-accesscontrol (org.apache.jackrabbit.vault.validation.spi.impl.AccessControlValidator),
jackrabbit-filter (org.apache.jackrabbit.vault.validation.spi.impl.AdvancedFilterValidator),
jackrabbit-properties (org.apache.jackrabbit.vault.validation.spi.impl.AdvancedPropertiesValidator),
jackrabbit-docviewparser (org.apache.jackrabbit.vault.validation.spi.impl.DocumentViewParserValidator),
jackrabbit-dependencies (org.apache.jackrabbit.vault.validation.spi.impl.DependencyValidator),
jackrabbit-emptyelements (org.apache.jackrabbit.vault.validation.spi.impl.EmptyElementsValidator),
jackrabbit-mergelimitations (org.apache.jackrabbit.vault.validation.spi.impl.MergeLimitationsValidator),
jackrabbit-oakindex (org.apache.jackrabbit.vault.validation.spi.impl.OakIndexDefinitionValidator),
jackrabbit-packagetype (org.apache.jackrabbit.vault.validation.spi.impl.PackageTypeValidator),
jackrabbit-nodetypes (org.apache.jackrabbit.vault.validation.spi.impl.nodetype.NodeTypeValidator)
[ERROR] ValidationViolation: "jackrabbit-packagetype: Package of type 'CONTAINER' must only contain sub packages of type
'APPLICATION, CONTENT, CONTAINER' but found sub package of type 'MIXED'!",
filePath=jcr_root/etc/packages/wcm-io/io.wcm.caconfig.editor.package-1.15.6.zip/META-INF/vault/properties.xml
[WARNING] ValidationViolation: "jackrabbit-packagetype: Package of type 'MIXED' is legacy. Use one of the other types instead!",
filePath=jcr_root/etc/packages/wcm-io/io.wcm.caconfig.editor.package-1.15.6.zip/META-INF/vault/properties.xml
```

I'm guessing there may have been a change in how strict the Jackrabbit packagetype validator is, which results in the package type used causing problems... Or perhaps I'm just doing something silly.

Perhaps there's a more straightforward way to get it installed neatly?
Penny for your thoughts.

Thanks,
Tomek

--- a/src/site/markdown/usage.md
+++ b/src/site/markdown/usage.md
@@ -13,18 +13,16 @@ See [Apache Sling Context-Aware Configuration documentation][sling-caconfig] for
 
 ### Installation
 
-In most cases you will deploy the configuration editor bundle `io.wcm.caconfig.editor` together with your application. In this case you should define your own template definition for it which controls where editor config pages can created (see next section).
+In production environments (including AEMaaCS), you should _only_ deploy configuration editor bundle `io.wcm.caconfig.editor` together with your application. In this case you should define your own template definition for it which controls where editor config pages can created (see next section).
 
-Alternatively you can deploy it using an AEM package which contains the config editor bundle together with a template definition which allows all paths below `/content`.
+For testing purpose an AEM package (ZIP) is provided which contains the config editor bundle together with a template definition which allows all paths below `/content`. This package is only for convenience installations on local AEM instances, it is not compatible with AEMaaCS. **Deploy only the bundle (JAR) on AEMaaCS.**
 
 |---|---|---|
 | [wcm.io Context-Aware Configuration Editor Bundle (JAR)](https://repo1.maven.org/maven2/io/wcm/io.wcm.caconfig.editor/) | [![Maven Central](https://img.shields.io/maven-central/v/io.wcm/io.wcm.caconfig.editor)](https://repo1.maven.org/maven2/io/wcm/io.wcm.caconfig.editor/) |
 | [wcm.io Context-Aware Configuration Editor AEM Package with Template Definition (ZIP)](https://repo1.maven.org/maven2/io/wcm/io.wcm.caconfig.editor.package/) | [![Maven Central](https://img.shields.io/maven-central/v/io.wcm/io.wcm.caconfig.editor.package)](https://repo1.maven.org/maven2/io/wcm/io.wcm.caconfig.editor.package/) |
 
 
-When you are using AEM 6.1 or 6.2 you have to additionally deploy the Apache Sling Context-Aware Configuration bundles (API, SPI, Impl) to AEM. In AEM 6.3 you have to update the Apache Sling Context-Aware Configuration SPI and Impl version to the latest version if you want to use Editor version 1.1 and upwards. See [Deploy and configure Context-Aware Configuration in AEM][deploy-configure-caconfig-in-aem] for details.
-
-In most cases the default persistence implementation provided by Apache Sling or AEM 6.3/6.4 is not enough for the needs of AEM applications. [wcm.io Context-Aware Configuration Extensions][wcmio-caconfig-extensions] provide additional implementations and features.
+In most cases the default persistence implementation provided by AEM out-of-the-box is not enough for the needs of AEM applications. [wcm.io Context-Aware Configuration Extensions][wcmio-caconfig-extensions] provide additional implementations and features.
 
 
 ### Defining the editor template

